converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Fhir/FhirService.cs

```diff
@@ -51,6 +57,30 @@ public Task<ImagingStudy> RetrieveImagingStudyAsync(Identifier identifier, Cance
         public Task<Endpoint> RetrieveEndpointAsync(string queryParameter, CancellationToken cancellationToken)
             => SearchByQueryParameterAsync<Endpoint>(queryParameter, cancellationToken);
 
+        /// <inheritdoc/>
```

Comment:
nit using System.Threading.Tasks; and remove the full name

Reply:
Updated, thanks!